### ISSUE TITLE
Extracting grid features with smaller image size

### ISSUE THREAD

**Issue by yuzcccc** [facebookresearch/grid-feats-vqa#4]
Thanks for sharing the code! 

I have a question when extracting the grid features: If I want to extract a fewer number of grid features (using a smaller image size like 448*448 which is mentioned in Table.4 of the paper), can I still use the provided pre-trained model or should I train a new model with smaller image sizes to ensure consistency?

**Comment by playerkk** [comment]
Thanks for your interest in our work. For the numbers reported in Table 4 of the paper, we trained separate models on Visual Genome using different image sizes.

**Comment by MIL-VLG** [comment]
Thanks for the reply

Just another question, will the number of grid features (196 vs 608) influence the used hyper-parameters (e.g., learning rate) to train the model?

**Comment by playerkk** [comment]
We used the same hyper-parameters for all grid features. But it's possible that optimal hyper-parameters may differ for different number of grid features.
